source/external-interface-design/external-interface-design.rst

```diff
@@ -548,9 +485,9 @@ KOMBIT Adgangstyring
 See `the seperate page for KOMBIT adgangsstyring <../kombit-adgangsstyring/kombit-adgangsstyring.html>`_
 
 
-.. |image1| image:: media/image5.png
+.. |image1| image:: media/image14.png
```

Comment:
We should probably give these some more descriptive names

Reply:
These image names is a pain in the *** when developing. 100% agreed that we could do something smarter here. Let's discuss it further.